Musicplayer.css

```diff
@@ -8,62 +8,32 @@ body{
 
 .grid-container{
 	display: grid;
-	grid-template-columns: 15% 10% 75%;
-	grid-template-rows: auto auto auto auto auto;
+	grid-template-columns: 15% 35% 50%;
```

Comment:
21/9: I change the setting for the grid template column and rows for it to fit with my aim